src/platform/src/app/(default)/(pages)/(layout-1)/settings/Tabs/Profile.jsx

```
@@ -144,79 +183,86 @@ const Profile = () => {
       setIsLoading(false);
       return;
     }
-    if (!loggedUser) return setIsLoading(false);
-    const userID = loggedUser._id;
+
+    const userID = session.user.id;
 
     // Create a cleaned version of userData without empty profilePicture
     const dataToUpdate = { ...userData };
     if (!dataToUpdate.profilePicture) {
       delete dataToUpdate.profilePicture;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)